librocksdb_sys/crocksdb/c.cc

```diff
@@ -436,6 +440,8 @@ struct crocksdb_compactionfilterfactory_t : public CompactionFilterFactory {
   void (*destructor_)(void*);
   crocksdb_compactionfilter_t* (*create_compaction_filter_)(
       void*, crocksdb_compactionfiltercontext_t* context);
+  bool (*should_filter_table_file_creation_)(
+      void*, crocksdb_tablefilecreationreason_t* reason);
```

Comment:
It is necessary to use pointer for an enum type?

Reply:
You are right there is no need. I replace pointers here in another [branch](https://github.com/MrCroxx/rust-rocksdb/commit/1c44be313eead76b3e7827627863272c2697a2f9). PTAL, and if that is fine I will merged it to this one.

Reply:
@MrCroxx Left a comment there. You should directly modify on current PR next time, to keep all discussions tracked.

Reply:
Okay, I'm not sure if this method is okay and I created the branch. I will merge this this branch first and work on the original branch.